### PR TITLE
Updates for FF and IE Driver Compatibility

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
An update for IE and FF driver compatibility.

### CODE REVIEW

File: packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js [hunk 1]
```diff
@@ -8,11 +8,7 @@ const ignoredA11y = {
 describe('Time Input Twelve Hour Mobile', () => {
   before(() => {
     browser.setViewportSize(Terra.viewports('tiny')[0]);
-    browser.execute(() => {
-      if (!window.ontouchstart) {
-        window.ontouchstart = 'true';
-      }
-    });
+    browser.execute('if (!window.ontouchstart) { window.ontouchstart = "true"; }');
```
Comment (JakeLaCombe) [inline packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: For every webdriver io test we have, does it do the test as a new page refreshed in the browser, or does it use the same page and just render different markup? Making sure we don't need to reset this when the test is complete.

Comment (emilyrohrbough) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: @JakeLaCombe Our current wdio setup refreshes the page after each test. This is something we need to remove to speed up testing. See https://github.com/cerner/terra-dev-site/issues/136

Comment (emilyrohrbough) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: That being said, I can add the removal now so we don't have to find this when we remove it 😄

Comment (JakeLaCombe) [reply packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js]: sounds good!

**Pull request merged**